diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -81,6 +81,7 @@ enum class Warnings(
     KDOC_NO_CONSTRUCTOR_PROPERTY(true, "2.1.1", "all properties from the primary constructor should be documented in a @property tag in KDoc"),
     KDOC_NO_CLASS_BODY_PROPERTIES_IN_HEADER(true, "2.1.1", "only properties from the primary constructor should be documented in a @property tag in class KDoc"),
     KDOC_EXTRA_PROPERTY(false, "2.1.1", "There is property in KDoc which is not present in the class"),
+    KDOC_DUPLICATE_PROPERTY(false, "2.1.1", "There is property in KDoc which is already present"),
```

Comment:
Please [proofread](https://writer.com/proofreading-checker/) this text. Should be at least "there's **a** property..."

Reply:
Fixed. However, proofread says, that there is `No proofing errors were found in your text.` in `There is property in KDoc which is already present`